release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3143,6 +3143,26 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * There is no log pruning available for azcopy (underlying tool running copy jobs) in Azure, so this might eventually lead to filling up a root device of the controller pod, and you have to manually clean this up. (link:https://issues.redhat.com/browse/OCPBUGS-35980[*OCPBUGS-35980*])
 
+* The limited live migration method stops when the `mtu` parameter of a `ConfigMap` object in the `openshift-network-operator` namespace is missing.
++
+In most cases, the `mtu` field of the `ConfigMap` object is created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` object might be absent.
++
+As a temporary workaround, you can manually create the `ConfigMap` object before starting the limited live migration process. For example:
```

Comment:
@stevsmit https://issues.redhat.com//browse/OCPBUGS-35316 is verified when the `mtu` parameter of a `ConfigMap` object in the `openshift-network-operator` namespace is missing. This is not an issue any more.